### ISSUE TITLE
Do we still need the add to favorite button in view favorite view?

### ISSUE THREAD

**Issue by lisazhong13** [HarryC493/CSC207Project#31]
Currently, if we view our favorite list and click on the restaurant, there's still an add to favorite button, which may be confusing. But since if we click the button, there will be a pop-up message saying restaurant is already in the favorite list, so I think this might not be a big issue

**Comment by choisar4** [comment]
You're right. I managed to remove the button effectively with Anthony's help. I have submitted a pull request under Remove Favourites Use Case, it just needs to pulled into main.
